src/index.html

```diff
@@ -11,7 +11,139 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    onsubmit="onSubmit()" method="POST"
+    >
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="Surname">Surname:</label>
+          <input
+          type="text" id="Surname"
+          autocomplete="off" required
+          >
```

Comment:
```suggestion
          <input
            type="text"
            id="Surname"
            autocomplete="off"
            required
          >
```
All attributes on the new lines and indentations